.gitignore

```diff
@@ -40,3 +40,6 @@ data
 
 # google cloud creds using by capistrano
 .google_creds.json
+
+# ignore current docker-compose.yml
+docker-compose.yml
```

Comment:
@recurator don't you also need to remove the current docker-compose.yml if you no longer need it? Otherwise you might end up with a strange case of a file that exists, but is ignored.